README.md

```diff
@@ -1,2 +1,140 @@
-# usaf-recruiting-pa
-USAF Recruiting Service Path Analysis
+# US Air Force Recruiting Service Path Analysis
+
+## Welcome!
+This repository contains documentation describing the Air Force Recruiting Service (AFRS) Path Analysis engagement. It is intended to be a place where project team members, stakeholders, and other interested parties can keep track of goals and resources as we work together. 
+
+## History
+
+### AFRS
+
+The mission of Air Education and Training Command (AETC) begins with the Air Force Recruiting Service (AFRS), with headquarters at JBSA-Randolph, Texas. The AFRS mission is to recruit quality men and women with the right skills, at the right time, in the right numbers to sustain the combat capability of the U.S. Air Force. 
+
+(adapted from http://www.aetc.af.mil/Air-Force-Recruiting-Service/)
+
+### Engagement
+
+The Department of Defense (DoD) has engaged with 18F in a path analysis to assess the current state of Air Force Recruiting Information Support System-Total Force (AFRISS-TF) system and develop a high-level vision for a more modular and mobile-friendly system.
+
+While 18F will not specify the future state system over the course of the eight-week effort, it will help set direction and promote alignment within AFRS around which aspects of a new system are most important to pursue.
+
+
+## Users and stakeholders
+
+AFRS comprises three regional groups and 27 squadrons with more than 1,200 recruiters assigned throughout the United States, England, Germany, Japan, Puerto Rico and Guam. The Air Force brings in nearly 24,000 active-duty enlisted accessions each year. The command is responsible for accessing 100 percent of the enlisted force, 90 percent of the service's medical officers, approximately 25 percent of the line officers (through Officer Training School) and 100 percent of Air Force chaplains.
+
+(from http://www.aetc.af.mil/Air-Force-Recruiting-Service/)
+
+
+### AFRS Project Management Office (PMO)
+
+The system is managed by a product owner and three other deputy program managers that represent the three components. 
```

Comment:
Specify the components here rather than later. Otherwise someone reading through this may not know what that word refers to.